packages/react/app/src/widget/Widget.tsx

```diff
@@ -0,0 +1,36 @@
+import { PropsWithChildren } from 'react';
+import { BaseWidget, WidgetModule, WidgetProps } from '@equinor/fusion-framework-react/widget';
+import useAppModules from '../useAppModules';
+import { Fusion } from '@equinor/fusion-framework-react';
+
+interface WidgetComponentProps<TProps extends WidgetProps> {
+    readonly props?: TProps;
+    readonly name: string;
+    readonly widgetVersion?: {
+        type: 'version' | 'tag';
+        value: string;
+    };
+}
+
+export const Widget = <TProps extends WidgetProps>({
+    name,
+    children,
+    props,
+    widgetVersion,
+}: PropsWithChildren<WidgetComponentProps<TProps>>) => {
+    const appModules = useAppModules<[WidgetModule]>();
```

Comment:
ummm explain this

Reply:
Did we not agree on widget used in applications shoudle only get app modules pased down? This wil require enableWiget in the app configuration.

Reply:
@odinr